### DISCUSSION TITLE
Version of python3-sphinx

### DISCUSSION THREAD

**Discussion by mark-bchoi** [bmwcarit/meta-ros#1221] [Q&A] [answered]
Hi, @jiaxshi 

python3-sphinx version is tied to 1.6.7 now.  

Is there any reason for that? python3-sphinx is provided newer version by open-embedded.
```
kirkstone : 4.4.0
scarthgap : 7.2.6
```

**Comment by jiaxshi** [comment, marked as answer]
i think it's the legay recipe inherits from initial meta-ros project. In early yocto and open-embedded distributions the python modules are not fully supported. And now they are support, I think it's good to use the latest python modules

**Reply by robwoolley** [reply]
Agreed.  I have seen similar problems with other recipes in meta-ros-common.   I think an audit would be a good idea.   @mark-bchoi please file a bug for this and we can address it.
